test/enforce-single-end-return.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Test for end-return rule
+ * @author Mitchell Van Der Hoeff <mitchellvanderhoeff@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require ('solium');
+
+var userConfig = {
+    rules: {
+        "security/enforce-single-end-return": "error"
+    }
+};
+
+describe ('[RULE] enforce-single-end-return: Acceptances', function () {
+
+    it ('should accept functions that have a single return statement at the end', function (done) {
+        var code = 'contract Foo { function foo () {uint256 x = 3; return;} }';
+        var errors;
+
+        errors = Solium.lint (code, userConfig);
+        errors.length.should.be.equal (0);
+
+        Solium.reset ();
+        done ();
+    });
+});
+
+describe ('[RULE] enforce-single-end-return: Rejections', function () {
+
+    it ('should reject functions that have a return statement not at the end', function (done) {
+        var code = 'contract Foo { function foo () {uint256 x = 3; if (x == 3) {return;} return;} }';
+        var errors;
+
+        errors = Solium.lint (code, userConfig);
+        errors.length.should.be.above (0);
```

Comment:
Don't test for "above 0". Instead have a test like `errors.should.have.size(<EXACT NUMBER OF ERRORS PRODUCED BY THS TESTED CODE>)`.
This means you might have to remove the for loop and test each code in the array separately, but that is the better practice.
We need to ensure that **given line produces exactly N amount of errors when only this rule is being applied**.